invite/src/types.ts

```diff
@@ -40,5 +40,5 @@ export interface Invite {
   repo: string;
   issue_number: number;
   action: InviteAction;
-  archived?: boolean;
+  archived?: boolean | number; // stored as a TINYINT in PostgresQL
```

Comment:
This is a little scary! The interface should stay as a boolean but be converted when retrieving or storing SQL data.

Reply:
I've dropped the `number` type and added explicit casting in the method that actually fetches from the DB.

Thanks for the link! Would it be alright with you if I move this to a declaration file in a separate PR? There are a handful of files that import it directly, and `Database` type could be moved into the type declaration file as well. I'd prefer to do that as a separate set of changes to minimize rebase headaches

Reply:
https://github.com/ampproject/amp-github-apps/pull/703

Reply:
Sounds good!

Reply:
Given that, is this PR LGTY?